docs/submission/validating-molecular-data.md

```diff
@@ -0,0 +1,443 @@
+---
+id: molecular-data-prep
+title: Preparing and Validating Molecular Data
+platform_key: DOCS_PREPARING_MOLECULAR_DATA
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+The DCC requires that metadata (necessary information describing molecular data files) is submitted at the same time as the molecular data to facilitate automated downstream analysis. This metadata is submitted to the ARGO metadata repository, called Song.  To prepare the molecular metadata, you will need to follow these steps: 
+
+1. Prepare metadata payloads according to the correct schema.
```

Comment:
Should we add the latest `sequencing_experiment` schema link here?

Reply:
we could, or we could point them to the documentation by saying `(described below)`.  I worry most users wont know how to read JSON schema?

Reply:
yeah I was confused where to find this "schema"

Reply:
suggest
`Prepare metadata payloads according to the normal and tumour examples shown below`

Reply:
1. Prepare metadata payloads according to the correct format (described below).